internal/pgbackrest/config.go

```diff
@@ -129,8 +131,7 @@ func CreatePGBackRestConfigMapIntent(ctx context.Context, postgresCluster *v1bet
 				postgresCluster.Spec.Backups.PGBackRest.Global,
 			).String()
 
-		if RepoHostVolumeDefined(postgresCluster) &&
-			collector.OpenTelemetryLogsOrMetricsEnabled(ctx, postgresCluster) {
+		if collector.OpenTelemetryLogsOrMetricsEnabled(ctx, postgresCluster) {
```

Comment:
So we're always adding OTel config to the configmap if OTel is enabled on this cluster? (Originally the plan was to only add Otel to repohost, but I guess we might also want to add it to cloud backup job pods?)

Reply:
This section of code changes is a little confusing because there was a redundancy removed... In the old version of the file, we called `RepoHostVolumeDefined` on line 99 and stored the result in `addDedicatedHost`. We then checked that value on line 118 before entering the `if` block. We then redundantly checked for a repo host again on line 132, but by calling `RepoHostVolumeDefined` again. I removed this redundant check...

So we still only check for OTel being enabled (now line 134) if a repo host is defined (checked on line 123).

Collecting logs for cloud backups was out of scope for this effort, so no changes to this yet.